src/IP.h

```diff
@@ -415,7 +415,11 @@ class IP_Hdr
 	uint16_t PayloadLen() const
 		{
 		if ( ip4 )
-			return ntohs(ip4->ip_len) - ip4->ip_hl * 4;
```

Comment:
Would be good to update the comment to spell out the special case when null can be returned.